### PR TITLE
Use NativeScript PubNub with kinvey-nativescript-sdk

### PR DESCRIPTION
**Pull request by thomasconner**
#### Description
Replace the `import 'pubnub'` with `import 'pubnub/lib/nativescript'` using webpack.

### CODE REVIEW

**Comment by chrismllr** [comment]
Hey @thomasconner -

Deleting your fork off the `pubnub` repo is breaking all npm installs of the latest version of the js-sdk `3.9.0`... 

```
error An unexpected error occurred: "https://codeload.github.com/thomasconner/javascript/tar.gz/07398c28df8bd2308e0c853818013134811c6de3: Request failed \"404 Not Found\"".
```

Is the plan that this PR will be merged ASAP?

**Comment by thomasconner** [comment]
Yes it is

**Comment by thomasconner** [comment]
@chrismllr I have recreated the fork which should resolve the issue.

**Comment by chrismllr** [comment]
Thanks @thomasconner - can confirm all is back to normal.

File: package-lock.json [hunk 1]
```diff
@@ -235,7 +235,8 @@
     "async": {
       "version": "1.5.2",
       "resolved": "https://registry.npmjs.org/async/-/async-1.5.2.tgz",
-      "integrity": "sha1-7GphrlZIDAw8skHJVhjiCJL5Zyo="
+      "integrity": "sha1-7GphrlZIDAw8skHJVhjiCJL5Zyo=",
```
[the remark this answers is not in the document]
Comment (thomasconner) [reply package-lock.json]: It seems like you should include the `package-lock.json` file in source control. https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

**Pull request merged**